doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -445,6 +472,12 @@ Libraries for networking
   * Updated:
 
     * The stack size of the MQTT connection monitoring thread can now be adjusted by setting the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD_STACK_SIZE` Kconfig option.
+    * The library now subscribes to a wildcard cloud-to-device (c2d) topic.
+      This enables the device to receive location services data on separate topics.
```

Comment:
```suggestion
      This enables the device to receive Location Services data on separate topics.
```

Generically referring to 'location data' would also be okay, but Location Services is a capitalized product name.